gulpfile.js

```diff
@@ -678,8 +678,6 @@ gulp.task('build-minify', function () {
     'src/angular-app/bellows/**/*.js',
     'src/angular-app/container/**/*.js',
     'src/angular-app/' + params.applicationName + '/**/*.js',
-    '!src/angular-app/languageforge/rapid-words/**', // ng2
-    '!src/angular-app/languageforge/review-suggest/**', // ng2
     '!src/angular-app/**/*.min.js',
```

Comment:
@megahirt This is an exclusion to stop the ng2 from being built in the same way as ng1.  So long as we remember to put the exclusion back in when we do ng2 apps removing the exclusion is fine.